src/main/java/uk/gov/hmcts/ccd/endpoint/std/CaseDetailsEndpoint.java

```diff
@@ -437,6 +453,7 @@ public CaseDetails createCaseEventForCitizen(
     @ApiOperation(value = "Get the pagination metadata for a case data search")
     @ApiResponses(value = {
         @ApiResponse(code = 200, message = "Pagination metadata for the given search criteria")})
+    @LogAudit(operationType = OperationType.SEARCH_CASE)
```

Comment:
I thing we can ignore this. We are already logging the search operation in searchCasesForCaseWorkers and searchCasesForCitizens. If we log here as well every user search would be logged twice which would be misleading. For one use operation I think it's better to have 1 log audit only.